app/src/components/market/sections/market_list/market_home_container.tsx

```diff
@@ -110,6 +111,7 @@ const wrangleResponse = (data: GraphMarketMakerDataItem[], networkId: number): M
 
 const MarketHomeContainer: React.FC = () => {
   const context = useConnectedWeb3Context()
+  const theme = useTheme()
```

Comment:
Yeah this hook seems to do the job but I still think we should avoid it since we already have one theme context with styled components and we should use that as much as we can. I would suggest try to implement everything inside styled components for example in history chart you can target stroke,tick and those properties if you just make one styled component out of YAxis for example....Or some workaround if possible